core/shared/src/main/scala/chartreuse/PlotModule.scala

```diff
@@ -94,13 +99,58 @@ trait PlotModule(numberFormat: NumberFormat) {
       val xTicksSequence = ticksToSequence(xTicks, scale, asX)
       val yTicksSequence = ticksToSequence(yTicks, scale, asY)
 
+      val xMajorTickToMinorTick: (ScreenCoordinate, Double, Int) => (
+          ScreenCoordinate,
+          DataCoordinate
+      ) = (screenCoordinate, interval, i) => {
+        val x = screenCoordinate.x - interval * i
+        (
+          ScreenCoordinate(x, 0),
+          DataCoordinate(x, 0)
+        )
+      }
+
+      val yMajorTickToMinorTick: (ScreenCoordinate, Double, Int) => (
+          ScreenCoordinate,
+          DataCoordinate
+      ) = (screenCoordinate, interval, i) => {
+        val y = screenCoordinate.y - interval * i
+        (
+          ScreenCoordinate(0, y),
+          DataCoordinate(0, y)
+        )
+      }
+
+      val toIntervalX: TicksSequence => Double = ticksSequence => {
+        val (screenCoordinate, _) = ticksSequence.head
+        val (screenCoordinate2, _) = ticksSequence.tail.head
+        (screenCoordinate2.x - screenCoordinate.x) / 4
```

Comment:
You seem to really really like putting [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) in your code. Please don't.

Reply:
Later, I intend to replace those with code which decides how many ticks to put. I should have mentioned this in the PR description.